libs/libimobiledevice/Makefile

```diff
@@ -8,7 +8,7 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libimobiledevice
-PKG_VERSION:=1.2.1
+PKG_VERSION:=custom-26373b33
```

Comment:
Patrząc tutaj: https://wiki.openwrt.org/doc/devel/packages#use_source_repository zdaje się, że przy budowaniu z prosto z gita nie podaje się PKG_VERSION. Źródła wydają się to potwierdzać (w tych Makefile'ach nie ma PKG_VERSION):
```
[jpc@jpc system] pwd
/Volumes/OpenWRT/package/system
[jpc@jpc system] ag PKG_MIRROR_HASH
procd/Makefile:17:PKG_MIRROR_HASH:=8d2c70fb0264a6548e39acc5ab1c44cec50fc550c4f931760b0796f8893c9b7f

fstools/Makefile:17:PKG_MIRROR_HASH:=8b483e752578683224245a6d77b3d7172b7cf7a8c3b959e21c47ff18aefc2464

lede-keyring/Makefile:12:PKG_MIRROR_HASH:=811ba79ba71925e949d2c690db7d7b031ac1dd965aa831ca9b6d9d70f5657254

usign/Makefile:10:PKG_MIRROR_HASH:=9499ed7e40889b364e446a428e185c40986b75087888bd7e1496542457a6dbaa

ubox/Makefile:10:PKG_MIRROR_HASH:=f58dfb9a9bb69c6303cb69bbd850b14bd29ec59ea240bac4063fd74a7fce64aa

opkg/Makefile:19:PKG_MIRROR_HASH:=1c9c21e56186345d0034d15051c1d9c5ebb5aa6658d6ae8886d8403207a914a3

ubus/Makefile:10:PKG_MIRROR_HASH:=6f46398279339dcc597965306275fe1272af384f8cb253ee8de2c68e366eed55

rpcd/Makefile:18:PKG_MIRROR_HASH:=4857497c88115defbf6add68a37975ed79e8f992e65d7d0df56cd29288dea379

uci/Makefile:18:PKG_MIRROR_HASH:=61f8cf52950edae851892ed0914dea8e1ab796bcefc095aa5dab28f672dc3301
```